__tests__/api.test.js

```diff
@@ -78,6 +79,52 @@ describe('GET /api/articles/:article_id', () => {
         })
     })
 })
+
+describe.only('GET /api/articles', () => {
+    test('should return 200 and an array where each article object has all the relevant properties', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            const articles = response.body.articles
+            expect(articles.length).toBe(13)
+            articles.forEach((article) => {
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(Number)
+                });
+            })
+        })
```

Comment:
we need another test for this endpoint that its default order is desc created_at - we can use jest sorted for this